analyses/molecular-subtyping-pathology/results/juvenile-xanthogranuloma-subtypes.tsv

```diff
@@ -1,5 +1,5 @@
 Kids_First_Biospecimen_ID	Kids_First_Participant_ID	sample_id	tumor_descriptor	broad_histology	short_histology	harmonized_diagnosis	Notes
 BS_2Z9JQDPV	PT_CAYBADQC	7316-2641	Initial CNS Tumor	Histiocytic tumor	JXG	Juvenile xanthogranuloma	Updated via OpenPBTA subtyping from pathology_free_text_diagnosis
-BS_4XPPZTGG	PT_EXJAEW1M	7316-188	Initial CNS Tumor	Histiocytic tumor	JXG	Juvenile xanthogranuloma	Updated via OpenPBTA subtyping from pathology_free_text_diagnosis
```

Comment:
Noting `7316-188` being removed here because `pathology_diagnosis` changed to `Craniopharyngioma`